src/surforama/app.py

```diff
@@ -157,10 +168,10 @@ def get_point_colors(self, points):
         point_values = self.volume[
             point_indices[:, 0], point_indices[:, 1], point_indices[:, 2]
         ]
+
         normalized_values = (point_values - point_values.min()) / (
-            point_values.max() - point_values.min()
+            point_values.max() - point_values.min() + np.finfo(float).eps
```

Comment:
This line was causing problems with the integer overflow. After converting tomograms to float, it should not be the case anymore.